run_tests.py

```diff
@@ -1,11 +1,6 @@
 from sqlalchemy.dialects import registry
 
-registry.register("db2", "ibm_db_sa.ibm_db", "DB2Dialect_ibm_db")
-registry.register("db2.ibm_db", "ibm_db_sa.ibm_db", "DB2Dialect_ibm_db")
-registry.register("db2.pyodbc", "ibm_db_sa.pyodbc", "DB2Dialect_pyodbc")
-registry.register("db2.zxjdbc", "ibm_db_sa.zxjdbc", "DB2Dialect_zxjdbc")
-registry.register("db2.pyodbc400", "ibm_db_sa.pyodbc", "AS400Dialect_pyodbc")
-registry.register("db2.zxjdbc400", "ibm_db_sa.zxjdbc", "AS400Dialect_zxjdbc")
+registry.register("ibmi", "sqlalchemy_ibmi.backend", "DB2Dialect_pyodbc")
```

Comment:
```suggestion
registry.register("ibmi", "sqlalchemy_ibmi.backend", "AS400Dialect_pyodbc")
```

Reply:
should we just remove run_test.py for now since the runner module is outdated and doesn't exist anymore in the sqlalchemy package

Reply:
sounds good. We can always go back to find it in git if needed, too